swap_meet/vendor.py

```diff
@@ -1,2 +1,139 @@
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    """
+    Represents a vendor with a list of inventory items
+    """
+    def __init__(self,inventory = None):
+        """
+        instatiates Vendor instances; has attribute 'inventory', which
+        initialized as an empty list if no value is passed in.
+        """
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self,item):
+        """
+        Adds item to a vendor's inventory list
+        """
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        """
+        Removes an item from a vendor's inventory list if the item is
+        present and returns the item; otherwise returns False
+        """
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
This is a small tweak, but it can often be a good idea to follow a pattern of starting with the negative guard clause as a conditional and then treating the rest of our function as the intended operation of the function. With that in mind, I would swap these checks so that we can have the "happy case" outside the conditional entirely!

```suggestion
        if item not in self.inventory:
            return False
            
       self.inventory.remove(item)     
       return item
```